src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+// Actions
+const ADDED = 'ADDED';
+const REMOVED = 'REMOVED';
```

Comment:
- [ ] As a good practice, you should try to scope the action names based on the feature they belong to. This helps when debugging more complex applications. You should follow the [docks pattern](https://github.com/erikras/ducks-modular-redux) for the value of each action type. You may do something like this.
`const ADDED = 'bookstore/books/ADDED'`